sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -686,7 +687,7 @@
             final boolean isFieldElementsIncluded = finalRecognizeBusinessCardsOptions.isFieldElementsIncluded();
             final FormRecognizerLocale localeInfo = finalRecognizeBusinessCardsOptions.getLocale();
             return new PollerFlux<>(
-                finalRecognizeBusinessCardsOptions.getPollInterval(),
+                DEFAULT_POLL_INTERVAL,
```

Comment:
Am I understanding this correctly that this makes the polling interval not configurable?

Reply:
it set the default value when creating poller, but we can override the value by using Poller's method `setPollInterval()`